main/acle.md

```diff
@@ -1713,6 +1713,15 @@ mechanisms such as function attributes.
 Pointer Authentication extension (FEAT_PAuth_LR) are available on the target.
 It is undefined otherwise.
 
+### Guarded Control Stack
+
+`__ARM_FEATURE_GCS_DEFAULT` is defined to `1` if the code generation is
```

Comment:
is set to

Reply:
Let's discuss about this proposed changes @philgee-oss . I will mark this as resolved for now.